pom.xml

```diff
@@ -37,6 +37,7 @@
         <module>conformance</module>
         <module>standardization</module>
         <module>dataModel</module>
+        <module>examples</module>
```

Comment:
With this added, please remove the `examples`  profile at the bottom (line 274)

Reply:
Done